### PR TITLE
Add documentation for new Repair days mapping configuration

### PR DESCRIPTION
**Pull request by markWarr**

### CODE REVIEW

File: docs/adoption.md [hunk 1]
```diff
@@ -259,6 +259,8 @@ To deploy the housing repairs api, you must first deploy `HousingRepairsScheduli
 | `STATE_KEY_NAME`                                  | The file path and name of your Terraform state file                                                           |
 | `STORAGE_CONTAINER_NAME_PRODUCTION`               | Storage container name for _Production_, e.g. `housing-repairs-online`                                        |
 | `STORAGE_CONTAINER_NAME_STAGING`                  | Storage container name for _Staging_, e.g. `housing-repairs-online-staging`                                   |
+| `REPAIR_DAYS_MAPPING_PRODUCTION`                  | Repair days mapping for communal repairs for _Staging_.                                                       |
+| `REPAIR_DAYS_MAPPING_STAGING`                     Repair days mapping for communal repairs for _Prodution_.                                                       |
```
Comment (Bhavesh-Patel) [inline docs/adoption.md:263]: Nitpick: Add missing `|`
```suggestion
| `REPAIR_DAYS_MAPPING_STAGING`                     | Repair days mapping for communal repairs for _Prodution_.                                                       |
```

Comment (Bhavesh-Patel) [inline docs/adoption.md:263]: Nitpick: Spelling correction
```suggestion
| `REPAIR_DAYS_MAPPING_STAGING`                     Repair days mapping for communal repairs for _Production_.                                                       |
```

File: docs/repairs-api/intro.md [hunk 2]
```diff
@@ -220,6 +222,24 @@ The following is a JSON schema for the allowed appointment slots data structure:
 }
 ```
 
+### Repair days Mapping
+This configuration is for the mapping between a repair's priority and the number of days in which it is expected to be completed. This mapping is used when reporting back to the user how long a repair is expected to take.
```
Comment (Bhavesh-Patel) [inline docs/repairs-api/intro.md:226]: This change includes
> expected to be completed

The phrasing in the prototype is:
> We will assess your request and a repair will be scheduled within the next 30 days.

I suspect the latter is correct, please confirm with Product Owner.
Documentation should be updated to reflect appropriately

File: docs/repairs-api/intro.md [hunk 3]
```diff
@@ -220,6 +222,24 @@ The following is a JSON schema for the allowed appointment slots data structure:
...
+
+The values of `REPAIR_DAYS_MAPPING_PRODUCTION` and `REPAIR_DAYS_MAPPING_PRODUCTION`should be in JSON format and be an array of objects with `Priority` and `NumberOfDays` defined, i.e.
```
Comment (Bhavesh-Patel) [inline docs/repairs-api/intro.md:228]: Nitpick: add a space
```suggestion
The values of `REPAIR_DAYS_MAPPING_PRODUCTION` and `REPAIR_DAYS_MAPPING_PRODUCTION` should be in JSON format and be an array of objects with `Priority` and `NumberOfDays` defined, i.e.
```

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]

File: docs/repairs-api/intro.md (hunk 2 above)
Comment (markWarr) [reply docs/repairs-api/intro.md:226]: Updated text to 'This configuration is for the mapping between a repair's priority and the number of days in which it is expected to be scheduled and completed. ' after conversation with Sonia.

**Review by Bhavesh-Patel — APPROVED** [review]

**Pull request merged**